pkg/operator/deploy/deploy.go

```diff
@@ -368,6 +369,28 @@ func (o *operator) CreateOrUpdate(ctx context.Context) error {
 	return nil
 }
 
+// CreateOrUpdateCredentialsRequest just creates/updates the ARO operator's CredentialsRequest
+// rather than doing all of the operator's associated Kubernetes resources.
+func (o *operator) CreateOrUpdateCredentialsRequest(ctx context.Context) error {
+	templ, err := template.ParseFS(embeddedFiles, "staticresources/credentialsrequest.yaml")
```

Comment:
If this path changes, will this break in some way?  

I guess our e2e will catch it since our e2e runs an update.  Just nervous about hardcoded strings in relation to path.

Reply:
Yep, E2E would break if the path were to change.

The only alternative I thought of to hardcoding the path was constructing the object in code (as opposed to reading in YAML from a file) and creating the CredentialsRequest over in `pkg/cluster` with the client-go dynamic client, but then we're duplicating the same CR content in different places, which seems worse than hardcoding a file path.